CHANGELOG.md

```diff
@@ -362,19 +402,19 @@ on the right-hand rule when exporting data in Json format
 
 ### Changed
 
-- GML encoding: Automatic conversion of polygon geometries to line geometries when there are no possibilities to encode a surface has been changed to produce a *LineString* for a *Polygon* and a *MultiLineString* for a *MultiPolygon*
```

Comment:
Just wondering - any specific reasons for these changes?

Reply:
No specific reason. And moreover no idea how it happened. 
I have change them back.